### PR TITLE
Change the alpine docker artifacts

### PR DESCRIPTION
**Pull request by Chathuratharushka**
This will fix [#7987](https://github.com/wso2-enterprise/financial-open-banking/issues/7987)

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: dockerfiles/alpine/obiam/README.md [hunk 1]
```diff
@@ -29,9 +30,9 @@ git clone https://github.com/wso2/docker-open-banking.git
...
-    > eg:- **Hosted locally**: `docker build --build-arg WSO2_OB_Accelerator_DIST_URL=http://localhost:8000/wso2-obiam-accelerator-3.0.0.tar.gz --build-arg WSO2_OB_KEYMANAGER_DIST_URL=http://localhost:8000/wso2is-extensions-1.2.10.tar.gz --build-arg WSO2_OB_CERTS_URL=http://localhost:8000/ob-cert.zip  -t wso2-obiam:2.0.0-alpine .` <br><br>
-      eg:- **Hosted remotely**: `docker build --build-arg WSO2_OB_Accelerator_DIST_URL=http://<public_ip:port>/wso2-obiam-accelerator-3.0.0.tar.gz --build-arg WSO2_OB_KEYMANAGER_DIST_URL=http://<public_ip:port>/wso2is-extensions-1.2.10.tar.gz --build-arg WSO2_OB_CERTS_URL=http://<public_ip:port>/ob-cert.zip  -t wso2-obiam:2.0.0-alpine .`
+    + `docker build --build-arg WSO2_OB_Accelerator_DIST_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> --build-arg WSO2_OB_KEYMANAGER_DIST_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> --build-arg OB_TRUSTED_CERTS_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> --build-arg WSO2_OB_KEYSTORES_URL=<URL_OF_THE_HOSTED_LOCATION/FOLDER_NAME> -t wso2-obiam:2.0.0-alpine .` <br>
```
Comment (KalanaDananjaya) [inline dockerfiles/alpine/obiam/README.md:33]: The tag is incorrect. Should be changed to 3.0.0

File: dockerfiles/ubuntu/obam/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This section defines the step-by-step instructions to build an [Ubuntu](https://
   > The hosted locations of artifacts will be passed as the build arguments when building the Docker image.<br>
   > 1. **WSO2_OB_Accelerator_DIST_URL** - Accelerator location
   > 2. **OB_TRUSTED_CERTS_URL** - Certificate zip archive location
-  > 3. **WSO2_OB_KEYSTORES_URL** - OBAM Keystores folder location
+  > 3. **WSO2_OB_KEYSTORES_URL** - OBAM Keystores folder location (https://github.com/wso2/docker-open-banking/raw/v3.0.0.6/dockerfiles/ubuntu/obam/obam-keystores)
```
Comment (KalanaDananjaya) [inline dockerfiles/ubuntu/obam/README.md:14]: new tag will be v3.0.0.7

Comment (KalanaDananjaya) [reply dockerfiles/ubuntu/obam/README.md:14]: Check for other places as well

File: dockerfiles/alpine/obam-uk/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,116 @@
+# ----------------------------------------------------------------------------------------
```
Comment (KalanaDananjaya) [inline dockerfiles/alpine/obam-uk/Dockerfile:1]: Since we have not tested uk toolkit for alpine images, lets not add them

**Review by KalanaDananjaya — CHANGES_REQUESTED** [review]

File: dockerfiles/alpine/obiam/README.md (hunk 1 above)
Comment (Chathuratharushka) [reply dockerfiles/alpine/obiam/README.md]: changed it https://github.com/wso2/docker-open-banking/pull/149/commits/89b83d64b61cb15abe1b1f4ea239a556a28f5fbc

File: dockerfiles/ubuntu/obam/README.md (hunk 2 above)
Comment (Chathuratharushka) [reply dockerfiles/ubuntu/obam/README.md]: changed it in every necessary places. https://github.com/wso2/docker-open-banking/pull/149/commits/89b83d64b61cb15abe1b1f4ea239a556a28f5fbc

File: dockerfiles/alpine/obam-uk/Dockerfile (hunk 3 above)
Comment (Chathuratharushka) [reply dockerfiles/alpine/obam-uk/Dockerfile]: done https://github.com/wso2/docker-open-banking/pull/149/commits/89b83d64b61cb15abe1b1f4ea239a556a28f5fbc

**Review by KalanaDananjaya — APPROVED** [review]

**Review by hasithakn — APPROVED** [review]

**Pull request merged**